services/github/github-api-provider.integration.js

```diff
@@ -4,70 +4,98 @@ const { expect } = require('chai')
 const serverSecrets = require('../../lib/server-secrets')
 const GithubApiProvider = require('./github-api-provider')
 
-describe('Github API provider with token pool', function() {
+describe('Github API provider', function() {
```

Comment:
This diff is gnarly, though the effect was moving the old text into a `context` and adding a second `context` for the  global token / `{ withPooling: false }` version.